File: 30 Days of Code/Day 6 Let's Review/Solution.cpp
// https://www.hackerrank.com/challenges/30-review-loop/problem

#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */  
    int testCase;    
    cin>>testCase;
    string name[testCase];
    for(int index=0; index<testCase; ++index){
        cin>>name[index];
    }
    for(int i=0; i<testCase; ++i){
        string evenIndex, oddIndex;
        for(int j=0; j<name[i].size(); ++j){            
            if(j%2 == 0){
                evenIndex += name[i][j];
            }
            else{
                oddIndex += name[i][j];
            }
        }
        cout<<evenIndex<<" "<<oddIndex<<endl;
    }
    return 0;
}

File: 30 Days of Code/Day 6 Let's Review/Solution.py
# https://www.hackerrank.com/challenges/30-review-loop/problem

# Enter your code here. Read input from STDIN. Print output to STDOUT

testCase = int(input())
names = []
for i in range(testCase):
    name = input()
    names.append(name)

for name in names:
    evenIndex = ""
    oddIndex = ""
    for i in range(len(name)):
        if i%2 == 0:   
            evenIndex += name[i]
        else:
            oddIndex += name[i]

    print(evenIndex, oddIndex)
